packages/qsreplace/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=qsreplace
+pkgver=3.0b053d2
+pkgrel=1
+pkgdesc='Accept URLs on stdin, replace all query string values with a user-supplied value, only output each combination of query string parameters once per host and path.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/tomnomnom/qsreplace'
+license=('MIT')
+depends=()
```

Comment:
Remove empty array.

Reply:
removed

Reply:
Please check/apply the new suggestions too.